CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
   * Update PPRiskMagnes to version 5.5.0 with privacy manifest changes and code-signing
   * This version of the PPRiskMagnes framework is dynamic
 * Require Xcode 15.0+ and Swift 5.9+ (per [Apple App Store requirements](https://developer.apple.com/news/upcoming-requirements/?id=04292024a))
+* Security
+  * Updated expiring pinned vendor SSL certificates
```

Comment:
Same comment as the v6 PR - lets add this above `BraintreePayPalDataCollector` and not nested under security so it's clear this applies to the entire SDK